src/utils/filterPeople.ts

```diff
@@ -0,0 +1,39 @@
+import { GenderFilters, Person, SearchQueries } from '../types';
+
+export function filterPeople(
+  people: Person[],
+  params: URLSearchParams,
+): Person[] {
+  let filtered = [...people];
+
+  if (params.has(SearchQueries.sex)) {
+    if (params.get(SearchQueries.sex) !== GenderFilters.All) {
+      filtered = filtered.filter(
+        person => person.sex === params.get(SearchQueries.sex),
+      );
+    }
```

Comment:
You are calling `params.get(SearchQueries.sex)` twice, which is unnecessary. You can store the result in a variable and use it in your `if` statement and filter function.